dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -3733,6 +3737,7 @@ def test_null_calculation_with_differently_sampled_cols(self):
         # The only null in those rows in second column in that subset are 5, 7
         # Therefore only 2 rows have null according to row_has_null_count
         self.assertEqual(0, profile.row_is_null_count)
+        self.assertTrue(profile.options.row_statistics.null_count.is_enabled)
```

Comment:
I think the counts intuitively speak to whether this is enabled. The tests in the options should cover this being set properly as well.

We just want to make sure when enabled, it does collect the values as expected.